.circleci/config.yml

```diff
@@ -19,9 +16,75 @@ jobs:
       - run:
           name: Run unit tests
           command: |
-            export PYTHONPATH=$SMAUG_HOME/build:$PYTHONPATH
+            export PYTHONPATH=$SMAUG_HOME:$PYTHONPATH
             make test-run
+      - run:
+          name: Build documentation
+          command: |
+            apt install -y doxygen
+            doxygen Doxyfile
```

Comment:
Does this mean every commit will now build the docs, though only those on the master branch will push to the docs repo?

Reply:
Yes. If doc building fails we should know about it.